src/Strategy/Cargo.hs

```diff
@@ -156,6 +201,69 @@ instance ToJSON CargoProject
 instance AnalyzeProject CargoProject where
   analyzeProject _ = getDeps
 
+data CargoPackage = CargoPackage
+  { unLicense :: Maybe Text.Text
+  , unLicenseFile :: Maybe Text.Text
```

Comment:
[nit]: Generally, `un`-prefixed names are reserved for newtypes, and only then when a field name can't really provide extra semantic meaning.  I'd prefer to give these normal field names.